### PR TITLE
updated code

### PR DESCRIPTION
**Pull request by cs-pranjali-munfan**
PR title: Updated connector

Description:
Updated actions and parameters name and description
Updated connector code as needed
Added sample playbooks
Updated logos

Impact: All actions

UTC:

Install the connector
Verified all actions with sample inputs
Verified check health
Verified the Connector sample playbooks installed.
verified output schema for all  actions

### CODE REVIEW

File: fidelis-edr/info.json [hunk 1]
```diff
@@ -0,0 +1,977 @@
...
+      "parameters": [
+      ]
+    },
+    {
+      "operation": "get_script_packages",
+      "title": "Get Script Packages",
+      "description": "Get a list of all script packages from the Fidelis Endpoint EDR server",
+      "enabled": true,
+      "category": "investigation",
+      "annotation": "get_schema",
```
Comment (cs-prerna-joshi) [inline fidelis-edr/info.json]:
```suggestion
      "annotation": "get_script_packages",
```

File: fidelis-edr/operations.py [hunk 2]
```diff
@@ -56,20 +75,23 @@ def make_api_call(self, endpoint=None, method='GET', data=None, params=None):
...
             headers = {'Authorization': "Basic " + b64_credential, 'Content-Type': 'application/json'}
-            response = requests.request('GET', url, headers=headers, verify=self.verify_ssl)
-            return json.loads(response.content)['data']['token']
+            response = self.make_api_call(endpoint=url, headers=headers, login_flag=True)
+            if response.get('data'):
+                return response.get('data').get('token')
+            else:
+                logger.exception(str(response))
+                raise ConnectorError(str(response.get('error', '')))
         except Exception as err:
```
Comment (cs-suyog-jinturkar) [inline fidelis-edr/operations.py]: We don't need to catch this exception, since we are raising it again

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comment

File: fidelis-edr/info.json [hunk 3]
```diff
@@ -0,0 +1,977 @@
...
+        }
+      ]
+    },
+    {
+      "operation": "get_script_packages_template",
+      "title": "Get Script Packages Template",
+      "description": "Get Script Package Template from the Fidelis Endpoint EDR server based on script ID you have provided.",
+      "enabled": true,
+      "category": "investigation",
+      "annotation": "get_schema",
```
Comment (cs-prerna-joshi) [inline fidelis-edr/info.json]:
```suggestion
      "annotation": "get_script_packages_template",
```

File: fidelis-edr/info.json [hunk 4]
```diff
@@ -0,0 +1,977 @@
...
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "placeholder": "e.g [\"CEFOutput\",\"LEEFOutput\"]"
+        },
+        {
+          "title": "Questions",
+          "type": "json",
+          "name": "questions",
+          "description": "",
```
Comment (cs-prerna-joshi) [inline fidelis-edr/info.json]: can we add description.

File: fidelis-edr/info.json [hunk 5]
```diff
@@ -0,0 +1,977 @@
...
+          "tooltip": "",
```
Comment (cs-prerna-joshi) [inline fidelis-edr/info.json]: add tooltip.

**Pull request merged**